org.osgi.util.feature/src/org/osgi/util/feature/Feature.java

```diff
@@ -35,35 +37,63 @@
     String getName();
 
     /**
-     * Get the description.
-     * @return The description.
-     */
+	 * Get the categories.
+	 * 
+	 * @return The categories.
+	 */
+	Collection<String> getCategories();
```

Comment:
Perhaps this should be `List` since there is an order in the document? I think `List` is the best mapping to a JSON array.

Reply:
Categories don't have an order but sets aren't supported in JSON, so I picked Collection to make the Java API closer to the semantics. However it's probably better to map the Java API closer to the JSON representation, so I've changed is as you suggested.